cpp/src/gandiva/precompiled/arithmetic_ops.cc

```diff
@@ -507,6 +507,16 @@ FLOOR(float32)
 FLOOR(float64)
 
 #undef FLOOR
+#define SQRT(TYPE) \
+  FORCE_INLINE     \
+  gdv_float64 sqrt_##TYPE(gdv_##TYPE in1) { return static_cast<gdv_##TYPE>(sqrt(in1)); }
```

Comment:
You may need to be given a context to handle errors, such as entering negative numbers